packages/frontend/src/components/common/ResourceView/ResourceActionHandlers.tsx

```diff
@@ -45,6 +47,7 @@ const ResourceActionHandlers: FC<ResourceActionHandlersProps> = ({
     onAction,
 }) => {
     const { projectUuid } = useParams<{ projectUuid: string }>();
+    const { data: spaces } = useSpaceSummaries(projectUuid, true, {});
```

Comment:
now that we include private spaces (if the user has the ability) do we want to have 2 separate views or a visual indicator like you've mentioned internally?

Reply:
yes, definitely! I'll create a sub-ticket for that. thanks for reminding me